core/src/main/resources/lib/hudson/editable-description.js

```diff
@@ -0,0 +1,15 @@
+(function(){
+  document.addEventListener("DOMContentLoaded", function() {
```

Comment:
This works if and only if the `#description-link` element is already present when the page is loaded.

I think that's always the case, but would welcome counter-examples.

Reply:
This was done by you.

Reply:
Correct. This comment is not negative feedback, but providing context for other reviewers.